test/tests/config_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'enigma/config'
+require 'minitest/autorun'
+
+class ConfigTest < MiniTest::Test
+  def test_parsesCorrectly
```

Comment:
```suggestion
  def test_parses_correctly
```
Sorry to be pedantic, any objections to following the conventions of the language?